controllers/cluster_controller_test.go

```diff
@@ -2668,6 +2668,61 @@ var _ = Describe("cluster_controller", func() {
 					})
 				})
 			})
+			When("using ssd-redwood-1-experimental", func() {
```

Comment:
This test is nested in the wrong `When` statement, currently it's part of `When("creating a cluster with RocksDB as storage engine", func() {` which is wrong, that generates a test case like this: `when creating a cluster with RocksDB as storage engine when using ssd-redwood-1-experimental when using 6.3.24 generations are not matching`.

This test is currently failing since the `IsStorageEngineSupported` is not updated.

Reply:
Sorry for this.